tests/Dfe.PlanTech.Web.E2ETests/README.md

```diff
@@ -9,19 +9,30 @@ Created using [Cypress](https://cypress.io)
 ## Setup
 
 1. Copy the `cypress.env.json.example` file and rename it to `cypress.env.json` in the root of the `Dfe.PlanTech.Web.E2ETests` folder
-2. Populate the variables within it
-3. Install the necessary packages by running `npm install`
-4. Run Cypress by running `npx cypress open --config "baseUrl=URL"` where URL is the same as in the variables mentioned below
+2. Populate the environment variables within it (guidance below).
+3. Install the necessary packages by running `npm install` from `Dfe.PlanTech.Web.E2ETests`.
+4. Run Cypress by running `npx cypress open --config "baseUrl=URL"` where URL is the same as in the variables mentioned below.
+
+Further setup is required to use the Dynamic Page Validator - see the dedicated README.
```

Comment:
This update is great, you could link the readme here as well

Reply:
Done